src/main/java/seedu/address/logic/commands/ImportCommand.java

```diff
@@ -69,47 +64,52 @@ public ImportCommand(Path importCsvFilePath) {
         JsonUserPrefsStorage userPrefStorage =
             new JsonUserPrefsStorage(projectRootPath.resolve("config.json"));
         this.storage = new StorageManager(jsonStorage, userPrefStorage);
-        haveDuplicatePersons = false;
     }
 
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
         requireNonNull(importCsvFilePath);
         requireNonNull(storage);
-        readCsvFile(model);
-        return new CommandResult(MESSAGE_SUCCESS);
+        int studentsImported = readCsvFile(model);
```

Comment:
Maybe since now readCsvFile returns an int, its name should be changed to getImportedStudents, to reflect its purpose better and to prevent ambiguity

Reply:
excellent point! I'll fix it asap